_includes/top-navigation.html

```diff
@@ -77,14 +77,23 @@
                               <li><a href="{{ site.baseurl }}/tools-support/tools/webtest.html">Tools</a></li>
                               <li><a href="https://forum.developer.concur.com/">Forum</a></li>
                               <li><a href="{{ site.baseurl }}/tools-support/support.html">Support</a></li>
-                              <li><a href="{{ site.baseurl }}/blog">Blog</a></li>
                               <li><a href="{{ site.baseurl }}/tools-support/release-notes/index.html">Release Notes</a></li>
                               <li><a href="{{ site.baseurl }}/tools-support/reference/custom-list-items.html">Reference</a></li>
                               <li><a href="{{ site.baseurl }}/newsletter-request.html">Newsletter</a></li>
                             </ul>
                         </li>
                         <!-- End Tools & Support -->
-
+                        <!-- Blog  -->
+                        {% if page.url == '/blog' %}
```

Comment:
Sorry, should have been more specific. Line 87 is where it needs to be /blog/ in order for the highlighting to work. At least that's the only way I could get it to work locally.